redis/src/main/scala/zio/redis/options/Strings.scala

```diff
@@ -2,6 +2,25 @@ package zio.redis.options
 
 trait Strings {
 
+  sealed trait StralgoCommand
+
+  object StralgoCommand {
+    sealed case class StralgoLCS(`type`: StralgoType) extends StralgoCommand
+  }
```

Comment:
I'd avoid the command suffix as it's heavily used for redis commands. Additionally, perhaps it makes sense to unify the two, e.g.:

```scala
sealed trait Algorithm

object Algorithm {
  case object StringLcs extends Algorithm
  case object KeysLcs extends Algorithm
}
```

Of course, you can (and should) further expand with parameters as discussed by you and @jxnu-liguobin .

Reply:
it looks really good to use the unsuffixed one. It is more easy to illustrate the relationship of subcommands

Reply:
I opted for the unsuffixed ones as that, to me, reflects the documentation/stucture of redis commands better.